src/routes/safe/components/Settings/ManageOwners/ReplaceOwnerModal/index.tsx

```diff
@@ -15,29 +14,35 @@ import replaceSafeOwner from 'src/logic/safe/store/actions/replaceSafeOwner'
 import { safeParamAddressFromStateSelector, safeThresholdSelector } from 'src/logic/safe/store/selectors'
 import { checksumAddress } from 'src/utils/checksumAddress'
 import { makeAddressBookEntry } from 'src/logic/addressBook/model/addressBook'
+import { sameAddress } from 'src/logic/wallets/ethAddresses'
+import { Dispatch } from 'redux'
```

Comment:
And here
https://github.com/gnosis/safe-react/blob/development/src/logic/safe/store/actions/types.d.ts